File: The Dog Walker/The Dog Walker/PetData.swift
//
//  PetData.swift
//  The Dog Walker
//
//  Created by Scott O'Hara on 6/17/17.
//  Copyright © 2017 Scott O'Hara. All rights reserved.
//

import Foundation


class PetData{
    
    //stored properties
    var petName: String
    var birthday: String
    var breed: String
    var meds: String
    var vaccine: String
    var specialInstructions: String
    var emergencyContact: String?
    var emergencyPhone: String?
    var vetName: String
    var vetPhone: String
    var petImage: String?
    var uid: String?
    var petKey: String?
    
    //init properties 
    init(petName: String, birthday: String, breed: String, meds: String, vaccine: String, specialInstructions: String, vetName: String, vetPhone: String) {
        
        self.petName = petName
        self.birthday = birthday
        self.breed = breed
        self.meds = meds
        self.vaccine = vaccine
        self.specialInstructions = specialInstructions
        self.vetName = vetName
        self.vetPhone = vetPhone
    }
    
    
}

File: TheDogWalker/app/src/main/java/com/ohara/thedogwalker/dataModel/PetData.java
package com.ohara.thedogwalker.dataModel;

import java.io.Serializable;



public class PetData implements Serializable {

    //stored properties
    public String petName;
    public String birthday;
    public String breed;
    public String meds;
    public String vaccines;
    public String specialIns;
    public String emergencyContact;
    public String emergencyPhone;
    public String vetName;
    public String vetPhone;
    public String petImage;
    public String uid;
    public String petKey;

    //firebase default constructor
    public PetData(){
    }

    //constructor
    public PetData(String petName, String birthday, String breed, String meds, String vaccines, String vetName, String vetPhone){

        this.petName = petName;
        this.birthday = birthday;
        this.breed = breed;
        this.meds = meds;
        this.vaccines = vaccines;
        this.vetName = vetName;
        this.vetPhone = vetPhone;
    }


}
